ble/Gap.h

```diff
@@ -140,10 +140,15 @@ class Gap {
         return (durationInMillis * 1000) / UNIT_1_25_MS;
     }
 
+    typedef FunctionPointerWithContext<TimeoutSource_t> TimeoutEventCallback_t;
+    typedef CallChainOfFunctionPointersWithContext<TimeoutSource_t> TimeoutEventCallbackChain_t;
```

Comment:
do we need a `TimeoutEventCallback_t` when we've got a `TimeoutEventCallbackChain_t`?